src/Extension/Tasks/Commands/CheckMutagenCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Extension\Tasks\Commands;
+
+use SprykerSdk\Sdk\Core\Domain\Entity\Message;
+use SprykerSdk\Sdk\Extension\ValueResolvers\PCSystemValueResolver;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\Entity\ConverterInterface;
+use SprykerSdk\SdkContracts\Entity\ErrorCommandInterface;
+use SprykerSdk\SdkContracts\Entity\ExecutableCommandInterface;
+use SprykerSdk\SdkContracts\Entity\MessageInterface;
+use Symfony\Component\Process\Process;
+
+class CheckMutagenCommand implements ExecutableCommandInterface, ErrorCommandInterface
+{
+    /**
+     * @return string
+     */
+    public function getCommand(): string
+    {
+        return '';
```

Comment:
Should it be `static::class` instead of empty string?

Reply:
it's ignoring doesn't matter. it's executable command